### PR TITLE
Dstara/wrap udf api

### PR DESCRIPTION
**Pull request by DimitrisStaratzis**
This PR wraps the autogenerated UDF API and offers return results in String, JSON, and Arrow format.

### CODE REVIEW

File: src/main/java/io/tiledb/cloud/CustomCode.txt [hunk 1]
```diff
@@ -46,3 +46,203 @@ public byte[] runSQLBytes(String namespace, SQLParameters sql, String acceptEnco
...
+/**
+ *
+ * send a UDF to run against a specified array/URI registered to a group/project
+ * @param namespace namespace array is in (an organization name or user&#39;s username) (required)
+ * @param array name/uri of array that is url-encoded (required)
+ * @param udf UDF to run (required)
+ * @param xPayer Name of organization or user who should be charged for this request (optional)
+ * @param acceptEncoding Encoding to use (optional)
+ * @param v2 flag to indicate if v2 array UDFs should be used, currently in beta testing. Setting any value will enable v2 array UDFs. (optional)
+ * @return ApiResponse&lt;File&gt;
```
Comment (shaunrd0) [inline src/main/java/io/tiledb/cloud/CustomCode.txt:148]: `ApiResponse<byte[]>`

File: src/main/java/io/tiledb/cloud/CustomCode.txt [hunk 2]
```diff
@@ -46,3 +46,203 @@ public byte[] runSQLBytes(String namespace, SQLParameters sql, String acceptEnco
...
+    return localVarResp.getData();
+}
+
+/**
+ *
+ * submit a multi-array UDF in the given namespace
+ * @param namespace namespace array is in (an organization name or user&#39;s username) (required)
+ * @param udf UDF to run (required)
+ * @param acceptEncoding Encoding to use (optional)
+ * @return ApiResponse&lt;File&gt;
```
Comment (shaunrd0) [inline src/main/java/io/tiledb/cloud/CustomCode.txt:189]: `ApiResponse<String>`

File: src/main/java/io/tiledb/cloud/CustomCode.txt [hunk 3]
```diff
@@ -46,3 +46,203 @@ public byte[] runSQLBytes(String namespace, SQLParameters sql, String acceptEnco
...
```
Comment (shaunrd0) [inline src/main/java/io/tiledb/cloud/CustomCode.txt:230]: `ApiResponse<byte[]>`

**Review by shaunrd0 — APPROVED** [review]
Some small nits for return types in documentation, otherwise looks good! :+1: I was able to run the all of the examples using this branch without any issues.

**Comment by DimitrisStaratzis** [comment]
> Some small nits for return types in documentation, otherwise looks good! 👍 I was able to run the all of the examples using this branch without any issues.

**Comment by DimitrisStaratzis** [comment]
> Some small nits for return types in documentation, otherwise looks good! 👍 I was able to run the all of the examples using this branch without any issues.

Done, thanks!

**Pull request merged**